src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4872,4 +4872,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureTuples" xml:space="preserve">
     <value>tuples</value>
   </data>
+  <data name="IDS_FeatureOutVar" xml:space="preserve">
+    <value>out var</value>
+  </data>
```

Comment:
This might not be the best name for public consumption. Perhaps "out variable declaration"?